decidim-proposals/app/commands/decidim/proposals/admin/copy_proposals.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # A command with all the business logic when an admin copies proposals from
+      # one component to another.
+      class CopyProposals < Rectify::Command
```

Comment:
I think it's be better if we named this `ImportProposals` (same for the controller, etc), what do you think?